eth/downloader/resultstore.go

```diff
@@ -88,6 +92,7 @@ func (r *resultStore) AddFetch(header *types.Header, snapSync bool) (stale, thro
 	if item == nil {
 		item = newFetchResult(header, snapSync)
 		r.items[index] = item
+		r.pendingCount++
```

Comment:
also update pendingBodyGauge?

Reply:
yeah, originally I thought that it would add too much noise here.  But the metrics are sent asynchronously so the gauge should be updated here.